app/views/ad_mailer/send_token.text.erb

```diff
@@ -0,0 +1,4 @@
+Poštovani,
+
+šaljemo link za <%= @action %> oglasa.
+<%= @url %>
```

Comment:
I'd suggest being a bit more verbose to avoid some potential confusion.

```suggestion
Na vaš zahtjev šaljemo vam link za <%= @action %> vašeg oglasa:
<%= @url %>

Molimo vas da ne šaljete ovaj mail dalje jer svatko tko ima ovaj link može mijenjati vaš oglas.

Ovaj e-mail je automatski poslan zato što ste vi ili netko drugi zatražio uređivanje oglasa objavljenog na https://www.potres-petrinja.hr/. Ukoliko to niste bili vi slobodno zanemarite ovaj e-mail. 

Hvala!
----------
https://www.potres-petrinja.hr/
```

Reply:
Great suggestion! I'd add to put `Vi`, `Vaš`, etc. capitalized as it's formal language.